integration_tests/test_global_set.py

```diff
@@ -0,0 +1,7 @@
+from lpython import i32
+
+s1: set[str] = {"a", "b", "c", "a"}
+s2: set[i32] = {1, 2, 3, 1}
+s3: set[tuple[i32, i32]] = {(1, 2), (2, 3), (4, 5)}
+
+assert len(s1) == 3
```

Comment:
Can we add asserts on lengths of other sets `s2` and `s3`? Also, is it possible to print these sets before the assert?

Reply:
Printing on `set` is not implemented till now.

Reply:
Ok, let's only have asserts for now.